lib/optimizely/decision_service.rb

```diff
@@ -359,8 +330,8 @@ def get_user_profile(user_id)
       # Returns Hash stored user profile (or a default one if lookup fails or user profile service not provided)
 
       user_profile = {
-        :user_id => user_id,
-        :experiment_bucket_map => {}
+        user_id: user_id,
```

Comment:
what's the reason to made this change.

Reply:
Its a rubocop change, convention is updated in Ruby >2.
Since it was not part of feature task, so reverted it.